PayloadPkg/OsLoader/OsLoader.c

```diff
@@ -1321,15 +1310,6 @@ PayloadMain (
   }
   AddMeasurePoint (0x4020);
 
-  //
-  // Load PreOsChecker
-  //
-  if (LoaderPlatformInfo != NULL) {
-    if (FeaturePcdGet (PcdPreOsCheckerEnabled) && (LoaderPlatformInfo->LdrFeatures & FEATURE_PRE_OS_CHECKER_BOOT)) {

```

Comment:
Can you remove the related code and definitions to PcdPreOsCheckerEnabled and FEATURE_PRE_OS_CHECKER_BOOT?

Reply:
Both PcdPreOsCheckerEnabled and FEATURE_PRE_OS_CHECKER_BOOT are only used in EHL platform code.
They will be removed from platform in a separate patch.

Reply:
Okay, then can we remove platform code first? or remove all together since EHL code will not work after merging this PR.

Reply:
EHL could work with this PR since this PR just remove the unused core code.
I think EHL could remove FEATURE_PRE_OS_CHECKER_BOOT. but PcdPreOsCheckerEnabled could still be used by the platform. Anyway, they could be addressed in another patch.

Reply:
No problem if someone cleans up the code. Thanks.